src/borg/archiver.py

```diff
@@ -957,7 +957,17 @@ def format_cmdline(cmdline):
                 print()
         return self.exit_code
 
-    def _info_repository(self, cache):
+    def _info_repository(self, repository, key, cache):
+        print('Repository ID: %s' % bin_to_hex(repository.id))
+        print('Cache: %s' % cache.path)
```

Comment:
maybe move "cache: ...", below the key file.
currently, "encrypted: ..." directly follows "cache: ..." so people could think the cache is encrypted.

rest lgtm.